lib/main.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'game'
+require_relative 'board'
+require_relative 'player'
+
+
+board = Board.new
+player1 = Player.new('Player_1', x = nil,board )
+player2 = Player.new('Player_2',x = player1.token,board)
+game = Game.new(player1,player2,board)
+game.switch_player
+board.populate_display
+until game.over?
```

Comment:
The `main.rb` is better just a file that initializes all needed objects. I suggest transferring lines 12-15 in a separate file (possibly a file containing a module [possibly game_module or something] that has a method called `game_cycle`). This module then you can call in `main.rb` as such:

`game_module.game_cycle`

This way your main.rb file will be much cleaner and easier to reason about.